src/wizard/dotnet/aspnet.md

```diff
@@ -21,7 +21,7 @@ Install-Package Sentry.EntityFramework -Version {{@inject packages.version('sent
 
 <div class="alert alert-info" role="alert"><h5 class="no_toc">Using .NET Framework prior to 4.6.1?</h5>
     <div class="alert-body content-flush-bottom">
-        <a href="https://docs.sentry.io/clients/csharp/">Our legacy SDK</a> supports .NET Framework as early as 3.5.
```

Comment:
I think here we should keep the absolute URL - if this is still used in a wizard the path without the URL wont work. Same for all wizard files.